ironfist/src/cpp/combat/creatures.h

```diff
@@ -110,6 +110,8 @@ enum CREATURE_FLAGS : __int16
 struct tag_monsterInfo
 {
   __int16 cost;
+  //__int16 secondary_cost_id;

```

Comment:
Woahwoahwoah -- wait, what?

I encourage you to actually play a week of the game with these changes. I expect you will get nonsense values for creature growth because the original game's code is reading the "growth" field, but it's moved. I recommend reviewing this wiki article: http://projectironfist.pbworks.com/w/page/57731943/Interacting%20with%20the%20Original%20Code